src/libraries/System.Formats.Tar/src/System/Formats/Tar/TarEntry.cs

```diff
@@ -551,7 +551,19 @@ await using (fs.ConfigureAwait(false))
                 }
             }
 
-            ArchivingUtils.AttemptSetLastWriteTime(destinationFileName, ModificationTime);
+            AttemptSetLastWriteTime(destinationFileName, ModificationTime);
+        }
+
+        private static void AttemptSetLastWriteTime(string destinationFileName, DateTimeOffset lastWriteTime)
```

Comment:
copied from Archiving.Utils.cs where it's also used by Zip as I didn't want to affect Zip in anything we want to port (although, as per discussion in the linked issue, I don't believe it would affect Zip)